yapapi/strategy/base.py

```diff
@@ -86,6 +87,23 @@ async def respond_to_provider_offer(
         """Respond with a modified `DemandBuilder` object to an offer coming from a provider."""
         raise NotImplementedError()
 
+    @abc.abstractmethod
+    async def invoice_accepted_amount(self, invoice: rest.payment.Invoice) -> Decimal:
+        """Return the amount we accept to pay for the invoice.
+
+        Current Golem implementation accepts the invoice if returned amount equals `invoice.amount`
+        and ignores the invoice if it is different. This will change in the future."""
+        raise NotImplementedError()
+
+    @abc.abstractmethod
+    async def debit_note_accepted_amount(self, debit_note: rest.payment.DebitNote) -> Decimal:
+        """Return the amount we accept to pay for the debit note.
+
+        Current Golem implementation accepts the debit note if returned amount equals a
```

Comment:
again, this comment doesn't belong here as there's no implementation here

Reply:
ditto